commands/move.js

```diff
@@ -63,34 +63,53 @@ function MovementSounds(socket, room, excludeDir) {
   });
 }
 
+const commands = [
+  /^go\s+(.+)$/i,
+  /^walk\s+(.+)$/i,
+  /^move\s+(.+)$/i,
+];
+
+const directions = [
```

Comment:
Don't split these into two arrays, use the same "patterns" array for all patterns.